code/datums/components/squeak.dm

```diff
@@ -60,8 +61,6 @@
 			var/obj/item/projectile/P = AM
 			if(P.original != parent)
 				return
-	if(istype(AM, /obj/effect/dummy/phased_mob)) //don't squeek if they're in a phased/jaunting container.
-		return
```

Comment:
Pretty sure you didn't mean to delete this

Reply:
thanks